### ISSUE TITLE
Suggestion: Multilanguage

### ISSUE THREAD

**Issue by Spelvi** [kapitainsky/RcloneBrowser#47]
Hello, thank you very much for all your work, a suggestion, it would be good translations to be multilanguage, including the Spanish that is my language.

**Comment by kapitainsky** [comment]
Yes it would be nice to be multilingual. I will look into making Rclone Browser able to use different language assets but then it will be up to the wider community to do translations.

**Comment by Spelvi** [comment]
> Yes it would be nice to be multilingual. I will look into making Rclone Browser able to use different language assets but then it will be up to the wider community to do translations.



Great, what good news! You will see how the community collaborates because you do very good work, RcloneBrowser is spectacular, very good work is what you do.

**Comment by nagleaidan** [comment]
I would be happy to help with Spanish translation once the ability to use different language assets is added

**Comment by dia3olik** [comment]
I would gladly help with the Italian translation ;-)

**Comment by kapitainsky** [comment]
Unfortunately it is much bigger task than I expected. I don't have time to do it.

**Comment by Spelvi** [comment]
A pity to read that, well, thanks alike.

**Comment by kapitainsky** [comment]
The biggest issue is that it is legacy application with core of it created by previous developer. It was not started with translation in mind and now would require rewriting tones of code.

Hopefully some Qt guru can take this task over - so even I am not able time wise to do it maybe it will happen one day.
